lib/rules/block-indentation.js

```diff
@@ -82,6 +82,16 @@ function isControlChar(char) {
 }
 
 module.exports = class BlockIndentation extends Rule {
+  static get meta() {
+    return {
+      description: 'requires blocks to be indented and aligned',
+      category: 'Stylistic Issues', // 'Deprecated Rules', 'Possible Error', 'Best Practices',
+      presets: { recommended: true },
+      url:
+        'https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rules/block-indentation.md',
+      fixable: false,
```

Comment:
I would like to exactly match the [eslint rule meta object](https://eslint.org/docs/developer-guide/working-with-rules#rule-basics-1):

```suggestion
      type: 'layout',
      docs: {
        description: 'requires blocks to be indented and aligned',
        category: 'Stylistic Issues', // 'Deprecated Rules', 'Possible Error', 'Best Practices',
        recommended: false,
        url: 'https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rules/block-indentation.md',
      },
      fixable: false,
      schema: [],
```